### PR TITLE
We're not talking about policy; mostly

### PR DESCRIPTION
**Pull request by martinthomson**
This is a hard needle to thread because policy controls are sometimes
unwanted or unexpected and that's not strictly positive.  Making a point
about exposure to unwanted or unexpected policy controls is part of the
message, but we also need to allow endpoints to make a deliberate choice
not to pass something.  I think that I managed that here, the SIP SBC
example is particularly good for that.

I'll follow up with some text on IP router alerts.

### CODE REVIEW

File: draft-iab-use-it-or-lose-it.md [hunk 1]
```diff
@@ -139,6 +139,11 @@ critical infrastructure (e.g., IP, BGP, DNS, or TLS).  It could even be
 necessary to come up with a new protocol design that uses a different method to
 achieve the same result.
 
+To some degree assumes that extensions are not deliberately blocked.  Some
```
Comment (tfpauly) [inline draft-iab-use-it-or-lose-it.md]: This first sentence doesn't seem to parse?

File: draft-iab-use-it-or-lose-it.md [hunk 2]
```diff
@@ -292,15 +297,17 @@ but sometimes advantageous, despite some significant downsides.  Consequently,
...
-middlebox participates varies from the basic functions that a router performs
-to full participation.  For example, a SIP back-to-back user agent (B2BUA)
-{{?B2BUA=RFC7092}} can be very deeply involved in the SIP protocol.
+spoofing of DNS records by middleboxes such as those that inject authentication
+or pay-wall mechanisms as an authentication and authorization check, which are
+now prevalent in hotels, coffee shops and business networks.
+
+Middleboxes are also protocol participants, to the degree that they are able to
+observe and act in ways that affect the protocol.  The degree to which a
+middlebox participates varies.  This can include middleboxes that apply their
```
Comment (tfpauly) [inline draft-iab-use-it-or-lose-it.md]: Cutting this sentence short ("The degree to which a middlebox participates varies.") makes it less clear in context. Can we extend it back to what it was?

Comment (tfpauly) [reply draft-iab-use-it-or-lose-it.md]:
```suggestion
middlebox participates varies.  Middleboxes can also apply their
```

**Comment by martinthomson** [comment]
Thanks Tommy, I've tweaked those and rebased the branch.

**Review by tfpauly — APPROVED** [review]

**Pull request merged**